src/astx/exceptions.py

```diff
@@ -0,0 +1,45 @@
+"""Module for Exceptions."""
+
+from typing import Optional
+
+from public import public
+
+from astx.base import (
+    NO_SOURCE_LOCATION,
+    ASTKind,
+    ASTNodes,
+    Expr,
+    ReprStruct,
+    SourceLocation,
+    StatementType,
+)
+from astx.tools.typing import typechecked
+
+
+@public
+@typechecked
+class ThrowStmt(StatementType):
+    """AST class for throw statements."""
+
+    message: Optional[Expr]
```

Comment:
```suggestion
    exception: Optional[Expr]
```

Instead of message, it should be a exception, because the exception will be a special object that will contain (or not) the message.

as we are not worried about the semantic yet ... so any expression could be ok

for tests, it could be just an instance of Identifier